Consul.AspNetCore.Test/Consul.AspNetCore.Test.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.0;netcoreapp3.1;net461</TargetFrameworks>
```

Comment:
As far as I understand this is a .Net Core project, in which case I don't think the 'net461' target is needed?